packages/proof-service/src/service/task-service.ts

```diff
@@ -19,7 +19,7 @@ class TaskService {
     let retries = 0;
     let delay = this.initialDelay;
 
-    while (retries < this.maxRetries) {
+    while (true) {
```

Comment:
Will k8s manager remove app process if we are in infinite loop?

Reply:
I'm forget to undo it. My bad. Just config while true to run on local machine